openlibrary/macros/LoanStatus.html

```diff
@@ -115,11 +115,8 @@
   $if secondary_action:
     $:macros.BookPreview(ocaid, linkback=not no_index)
 
-$elif ('eligibility' in doc) or (check_sponsorship and not ocaid and not availability.get('is_lendable')):
-  $ sponsorship = doc.get('eligibility') if 'eligibility' in doc else qualifies_for_sponsorship(doc, donate_only=True)
-  $if sponsorship.get('is_eligible'):
-    <a href="$(sponsorship['sponsor_url'])"
-       class="cta-btn cta-btn--sponsor dialog--open"
+$elif ('eligibility' in doc) or (check_sponsorship and not ocaid and qualifies_for_sponsorship(doc, donate_only=True).get('is_eligible')):
+    <a class="cta-btn cta-btn--sponsor dialog--open"
```

Comment:
```suggestion
    <a class="cta-btn cta-btn--shell cta-btn--sponsor dialog--open"
```
Button CTA unreadable without `cta-btn--shell` class:

![Screenshot from 2022-06-28 14-27-57](https://user-images.githubusercontent.com/28732543/176256620-619658c5-4f8a-448e-ae7a-512541a7a90e.png)